### ISSUE TITLE
Use transparent color for scrollbar

### ISSUE THREAD

**Issue by isidorn** [arcticicestudio/nord-visual-studio-code#22]
The current way the scroll bar hides all the decorations in the right margin.
Please use a transparent color like all other themes

**Comment by cta22** [comment]
Same problem with minimap in insiders edition.

**Comment by arcticicestudio** [comment]
Thanks for your report 👍 
This has already been fixed in #13, I'll try to deploy it next weekend.

**Comment by isidorn** [comment]
@arcticicestudio thanks. In the meantime I will just steal your colors and put them as custom colorization in my workbench settings
```
"scrollbarSlider.activeBackground": "#434c5e99",
"scrollbarSlider.background": "#3b4252aa",
"scrollbarSlider.hoverBackground": "#434c5e99",
```

**Comment by arcticicestudio** [comment]
Sorry for the delay!
It's shipped with #13.
